lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFDotNetClient.cs

```diff
@@ -153,6 +133,88 @@ public IJobSubmissionResult SubmitAndGetJobStatus(IJobSubmission jobSubmission)
             throw new NotSupportedException();
         }
 
+        /// <summary>
+        /// Gets the path of the submission package without submitting the job.
+        /// Only available in YarnREEFDotNetClient because implementation is YARN specific.
+        /// </summary>
+        [Unstable("0.14")]
+        public string GetSubmissionPackage(IJobSubmission jobSubmission, string applicationId)
```

Comment:
Also, the contents of our submission package depends on the `applicationId` in the YARN case.